genie-common/src/main/java/com/netflix/genie/common/dto/JobStatusMessage.java

```diff
@@ -0,0 +1,72 @@
+/*
+ *
+ *  Copyright 2015 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+package com.netflix.genie.common.dto;
+
+/**
+ * Constant strings for status message attached to a job after it terminates.
+ *
+ * @author mprimi
+ */
+public final class JobStatusMessage {
```

Comment:
It's possible we could make these eventually part of the JobStatus enum itself as another field if we were to expand the job status enums to more states. Would remove need for another class.